### ISSUE TITLE
Error in `search_users` only for a specific query

### ISSUE THREAD

**Issue by aedobbyn** [ropensci/rtweet#367]
Hey there -- great package!

I ran across an error which I think is occurring here https://github.com/ropensci/rtweet/blob/2fab11866e368d7c6d1525c82dc34a60939c9e6f/R/tweets_and_users.R#L151.

Not exactly sure what's going on because I've only noticed this error for this specific search query. It looks like the object is a list where the first element is a dataframe and the second two are null. Will update if I can dig into this more. I'm using version 0.6.9.

``` r
rtweet::search_users("joseph r yost")
#> Searching for users...
#> Error in `[[<-.data.frame`(`*tmp*`, i, value = c(58248669L, 210516776L, : replacement has 4 rows, data has 2
```

<sup>Created on 2019-11-01 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by mkearney** [comment]
I believe this has been fixed in newest version (`0.7.0`), which I will be sending to CRAN today!

``` r
> rtweet::search_users("joseph r yost")
Searching for users...
Finished collecting users!
# A tibble: 2 x 90
  user_id status_id created_at screen_name text  source display_text_wi… reply_to_status… reply_to_user_id reply_to_screen…
  <chr>   <chr>     <chr>      <chr>       <chr> <chr>             <int> <chr>            <chr>            <chr>           
1 582486… 58248669  NA         josephyost  NA    NA                   NA NA               NA               NA              
2 210516… 210516776 NA         JoseYostce… NA    NA                   NA NA               NA               NA              
# … with 80 more variables: is_quote <lgl>, is_retweet <lgl>, favorite_count <int>, retweet_count <int>, quote_count <int>,
#   reply_count <int>, hashtags <list>, symbols <list>, urls_url <list>, urls_t.co <list>, urls_expanded_url <list>,
#   media_url <list>, media_t.co <list>, media_expanded_url <list>, media_type <list>, ext_media_url <list>,
#   ext_media_t.co <list>, ext_media_expanded_url <list>, ext_media_type <chr>, mentions_user_id <list>,
#   mentions_screen_name <list>, lang <chr>, quoted_status_id <chr>, quoted_text <chr>, quoted_created_at <dttm>,
#   quoted_source <chr>, quoted_favorite_count <int>, quoted_retweet_count <int>, quoted_user_id <chr>,
#   quoted_screen_name <chr>, quoted_name <chr>, quoted_followers_count <int>, quoted_friends_count <int>,
#   quoted_statuses_count <int>, quoted_location <chr>, quoted_description <chr>, quoted_verified <lgl>,
#   retweet_status_id <chr>, retweet_text <chr>, retweet_created_at <dttm>, retweet_source <chr>,
#   retweet_favorite_count <int>, retweet_retweet_count <int>, retweet_user_id <chr>, retweet_screen_name <chr>,
#   retweet_name <chr>, retweet_followers_count <int>, retweet_friends_count <int>, retweet_statuses_count <int>,
#   retweet_location <chr>, retweet_description <chr>, retweet_verified <lgl>, place_url <chr>, place_name <chr>,
#   place_full_name <chr>, place_type <chr>, country <chr>, country_code <chr>, geo_coords <list>, coords_coords <list>,
#   bbox_coords <list>, status_url <chr>, name <chr>, location <chr>, description <chr>, url <lgl>, protected <lgl>,
#   followers_count <int>, friends_count <int>, listed_count <int>, statuses_count <int>, favourites_count <int>,
#   account_created_at <dttm>, verified <lgl>, profile_url <chr>, profile_expanded_url <chr>, account_lang <lgl>,
#   profile_banner_url <lgl>, profile_background_url <chr>, profile_image_url <chr>
> packageVersion("rtweet")
[1] ‘0.7.0’
```

**Comment by aedobbyn** [comment]
Amazing, thanks!!
